src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -756,6 +759,32 @@ private async ValueTask SendEndStreamAsync(int streamId)
             }
         }
 
+        private async ValueTask SendWindowUpdateAsync(int streamId, int amount)
```

Comment:
As with my comments on the previous PRs, there's no benefit in this case to using ValueTask, but it's probably not hurting anything measurably.